protostar/cli/misc.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+from typing import List
+
+
+def resolve_cairo_targets(targets: List[str]) -> List[Path]:
```

Comment:
could we reuse globbing logic used by the test runner here?

Reply:
i reused the logic from the format command

Reply:
OK, that's a great starting point 😄 

do as you wish, you can resolve this comment